devsite/devsite/settings.py

```diff
@@ -78,49 +82,70 @@
     },
 ]
 
-WSGI_APPLICATION = 'devsite.wsgi.application'
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/1.8/howto/static-files/
+
+# Static content
+STATIC_URL = '/static/'
 
+STATIC_ROOT = os.path.join(DEVSITE_BASE_DIR, "staticfiles")
+
+STATICFILES_FINDERS = [
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+]
+
+WSGI_APPLICATION = 'devsite.wsgi.application'
 
 # Database
 # https://docs.djangoproject.com/en/1.8/ref/settings/#databases
 
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'NAME': os.path.join(DEVSITE_BASE_DIR, 'figures-db.sqlite3'),
+        'USER': '',
+        'PASSWORD': '',
+        'HOST': '',
+        'PORT': '',
     }
 }
 
+LOCALE_PATHS = [
+   os.path.join(PROJECT_ROOT_DIR, 'figures', 'conf', 'locale')
+]
 
-# Internationalization
-# https://docs.djangoproject.com/en/1.8/topics/i18n/
-
-LANGUAGE_CODE = 'en-us'
-
-TIME_ZONE = 'UTC'
-
-USE_I18N = True
-
-USE_L10N = True
-
-USE_TZ = True
-
-
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.8/howto/static-files/
 
-STATIC_URL = '/static/'
+LOGIN_REDIRECT_URL = '/'
 
 # Open edX settings
 
 COURSE_ID_PATTERN = r'(?P<course_id>[^/+]+(/|\+)[^/+]+(/|\+)[^/?]+)'
 
-# Need to create WEBPACK_LOADER dict before importing edx_figures settings
-
-WEBPACK_LOADER = {}
-
-from figures.settings import FIGURES
-
-
+# For initial testing. Later we want to enforce authorization
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': [
+        'rest_framework.permissions.AllowAny',
+    ],
+    'DEFAULT_RENDERER_CLASSES': (
+        'rest_framework.renderers.JSONRenderer',
+        'rest_framework.renderers.BrowsableAPIRenderer',
+    )
+}
 
 
+# It expects them from the project's settings (django.conf.settings)
+WEBPACK_LOADER = {}
+CELERYBEAT_SCHEDULE = {}
+
+# Declare values we need from server vars (e.g. lms.env.json)
```

Comment:
Does the empty dict mean that Figures does not necessarily require anything? Or is this documented elsewhere?

Reply:
To keep the figures include fragment identical to that used in real Open edX, I have the empty dict since `ENV_TOKENS` are part of the edx-platform settings.

No defaults are required to be set in `lms.env.json` as `figures.settings` defines the defaults. The `ENV_TOKENS` settings are for defaults overrides and likely in the future for declaring plugins.

Since you raised the question, I'll elaborate on the doc line here